ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/attestation/ValidateableAttestation.java

```diff
@@ -38,12 +38,12 @@
   private final Supplier<Bytes32> hashTreeRoot;
   private final AtomicBoolean gossiped = new AtomicBoolean(false);
   private final boolean producedLocally;
+  private final OptionalInt receivedSubnetId;
```

Comment:
ok to lose `volatile` here?

Reply:
Yep, a `final` field can't change so can't be `volatile`.

Reply:
fair enough